spec/api-browser-window-spec.js

```diff
@@ -1204,6 +1204,17 @@ describe('BrowserWindow module', () => {
         })
       })
 
+      it('should focus on window correctly', () => {
+        w.destroy()
+        w = new BrowserWindow({
+          webPreferences: {
+            sandbox: true
+          }
+        })
+        w.focus()
+        assert(w.isFocused())
```

Comment:
I'm not sure this is testing the issue this patch fixes—does this test fail if you comment out the change you made to atom_api_web_contents.cc?